config.lua.dist

```diff
@@ -122,9 +122,13 @@ rateLoot = 2
 rateMagic = 3
 rateSpawn = 1
 
--- Monsters
+-- Monster Despawn Config
+-- despawnRange is the amount of floors a monster can be from its spawn position
+-- despawnRadius is how many tiles away it can be from its spawn position
+-- removeOnDespawn will remove the monster if true or teleport it back to its spawn position if false
 deSpawnRange = 2
 deSpawnRadius = 50
+removeOnDespawn = true
```

Comment:
IMHO this should default to false to keep previous mechanism.

Reply:
but previous mechanism was "wrong" and also with deSpawnRadius set to 50 I bet nobody would ever reach this anyway unless intentionally trying to lure a monster, I had to change mine to 20 because unless you are in a real map your players would hardly notice a change

Reply:
leaving it as true because it is the new default setting

Reply:
Wrong according to what? This is an MMORPG server emulator, not a cipbia clone 🤔